src/lib/dcob.rb

```diff
@@ -6,14 +6,33 @@
 require "pp"
 require "openssl"
 require "toml"
+require "rack"
+require "prometheus/client"
+require "prometheus/client/rack/collector"
+require "prometheus/client/rack/exporter"
 
 module Dcob
   class Server < Sinatra::Base
+    use Rack::Deflater, if: ->(env, status, headers, body) { body.any? && body[0].length > 512 }
+    use Prometheus::Client::Rack::Collector
+    use Prometheus::Client::Rack::Exporter
+
+    def initialize(app = nil)
+      super
+      prometheus = Prometheus::Client.registry
+      @no_dco_signoff = prometheus.counter(:no_dco_signoff, "The count of PRs with no DCO sign off")
+      @dco_signoff = prometheus.counter(:dco_signoff, "The count of PRs with a DCO sign off")
```

Comment:
"count of PRs" is probably misleading text. The counter will go up for every commit on a PR (one or more) for every update to a PR (new PR, pushed PR, force-pushed PR).

As implemented, maybe "count of commits"?